src/promptflow-evals/promptflow/evals/synthetic/_constants.py

```diff
@@ -0,0 +1,16 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+
+class SupportedLanguages:
+    """Supported languages for evaluation, using ISO standard language codes."""
+
+    Spanish = "es"
+    Italian = "it"
+    French = "fr"
+    German = "de"
+    SimplifiedChinese = "zh-cn"
+    Portuguese = "pt"
+    Japanese = "ja"
+    English = "en"
```

Comment:
should use this one. 
https://www.andiamo.co.uk/resources/iso-language-codes/

en-US and en-GB will act different.

Reply:
In my experience, Lang on GB differs from US. It can be spell difference or vocabularies too.
Example:
For vocabs:
apartments vs flats
Vacations vs holidays
For spells:
Color vs colours
Labor vs labour

Are these relevant here?

Reply:
english isn't passed to any backend service to make a difference.
 The `AdversarialSimulator` adds a line to the end of the adversarial prompt only if a non english T1 language is selected.

Reply:
Also, @WaqasJaved02 I had pointed out the importance of regional language differences in a meeting a few weeks ago, but unless we get indication that users want more specificity, it's safe to use the standard language. "en" is meant to be intelligible by all English speakers (same with Spanish, for example) - and if users eventually want more nuance, we can easily add "en-us", "en-gb", "en-au", etc.